docs/Configuring-Orion/Disaster-Recovery.md

```diff
@@ -4,18 +4,15 @@ description: Disaster recovery strategies
 # Disaster Recovery
 
 !!! danger
-    If the database is deleted or corrupted, all payloads for the node are lost. You cannot recover a lost database without a backup.
+    If the Orion database is deleted or corrupted, all private transaction payloads for the node are lost. 
+    You cannot recover a lost database without a backup.
 
-Orion stores all payload information in an internal database. The database location is specified by the 
-`workdir` and `storage` properties in the [configuration file](Configuration-File.md).
+Orion supports using relational databases PostgreSQL and Oracle. To enable disaster recovery for Orion, 
```

Comment:
Maybe add more are planned on the roadmap?

Reply:
We don't do this in the doc generally unless there's a really strong reason to (ie, there's a big limitation that we want to highlight is about to be fixed or if a choice they make now will limit their choices in regards to a planned feature)